CRM/Civixero/Job/Job.mgd.php

```diff
@@ -10,13 +10,14 @@
     'update' => 'never',
     'params' => [
       'version' => 3,
-      'name' => 'CiviXero Contact Push Job',
```

Comment:
Changing these names will invalidate existing managed entities - with the reconcile removing / recreating them - feels like it will be a pain to deal with the fallout for an arguable aestethic improvement